getting-started/basicapp/README.md

```diff
@@ -0,0 +1,362 @@
+# Building, Testing, and Optimizing Code on the Google Cloud
+This example demonstrates a web app that sends messages from a user to her or
+his friends. It demonstrates the use of Google [Cloud
+Build](https://cloud.google.com/cloud-build/) for building and testing portable
+Go code with an application that can be run locally, in unit tests, in 
+[App Engine Flex](https://cloud.google.com/appengine/docs/flexible/), a plain
+Docker container, and in Kubernetes. The app is instrumented for 
+[Stackdriver Profiler](https://cloud.google.com/profiler/) and 
+[Stackdriver Trace](https://cloud.google.com/trace/).
+
+
+## Project Setup
+The prerequisites for running the example locally are 
+[Go](https://golang.org/dl/), the 
+[gcloud](https://cloud.google.com/sdk/gcloud/) command line tool, and Docker
+community edition. If you prefer not to install gcloud and Docker locally you
+can use the [Cloud Shell](https://cloud.google.com/shell/docs/) instead. 
+
+To run this example, you will need to create a Google Cloud project with billing
+enabled. You will also need to enable the Cloud SQL API, Cloud SQL Admin API,
+the Stackdriver Profiling API, and the Stackdriver Trace API.
```

Comment:
Might be good to link to a cloud.google.com page with how to do these, plus direct links to enable the APIs.

Reply:
Added